src/pydocstyle/checker.py

```diff
@@ -111,7 +111,7 @@ class ConventionChecker:
         # Begins with 0 or more whitespace characters
         r"^\s*"
         # Followed by 1 or more unicode chars, numbers or underscores
-        # The above is captured as the first group as this is the paramater name.
+        # The below is captured as the first group as this is the parameter name.
```

Comment:
This was a drive-by fix.  It appears that the comment was moved at some point in the past and the directional word was not changed; plus the spelling correction.